ExtensionManifest.md

```diff
@@ -0,0 +1,88 @@
+- `namePrefix` <[String]> (Optional) Prefix used to group and differentiate UIX fields and statistics from others in Ringtail. This value is required if fields or statistics are specified. **NOTE: This value cannot be changed once set.**
+- `fields` <[Array]<[Field](#fields)>> (Optional) Fields to create in each case the UIX is assigned to.
+- `statistics` <[Array]<[Statistic](#statistics)>> (Optional) Statistics counters to create in each case the UIX is assigned to.
+
+## Fields
+Providing field settings in an extension manifest allows Ringtail to automatically create fields in cases where it is installed. This is a great way to expose UIX states to Ringtail, allowing users to search, list, and view these fields.
+
+- `id` <[Number]> Identifier for this field in the manifest, used to lookup its concrete identifier for coding.
+- `name` <[String]> Name of the field. This will be prefixed with the `namePrefix` value in Ringtail.
+- `type` <[String]> Type of data the field will store. Must be one of:
+  - `YesNo` - Simple boolean value
+  - `Number` - Numeric value in the range [-99999999999.9999, 99999999999.9999] with at most four decimal digits
+  - `Date` - Date value in ISO 8601 UTC format
+  - `Text` - Simple string values with at most 255 characters
+  - `Memo` - Larger string values supporting simple HTML markup with at most 4095 characters
+  - `PickList` - Value picked from a provided list of choices
+- `isOneToOne` <[Boolean]> (Default `true`) True to only allow a single value per document, otherwise multiple, unique values may be coded. `YesNo` fields require `isOneToOne=true`.
+- `excludeFromSearch` <[Boolean]> (Default `false`) True to suppress this field from Ringtail's advanced search builder.
+- `excludeFromListColumns` <[Boolean]> (Default `false`) True to suppress this field from documents list columns.
+- `excludeFromImport` <[Boolean]> (Default `false`) True to prevent this field from receiving imported data.
+- `excludeFromSecurity` <[Boolean]> (Default `false`) True to suppress this field from the field security UI.
+- `codingSettings` <[String]> (Default `MassCode`) Maximum-allowed functionality the field may be granted for the coding UI. Must be one of:
+  - `Hidden` - Does not appear
+  - `Read` - Appears but is read-only
+  - `Write` - Available for single coding
+  - `MassCode` - Available for bulk coding
+- `items` <[Array]<[FieldItem](#field-items)>> Choices avaiable for `PickList` fields.
+
+## Field Items
+Field items are pre-defined choices for `PickList` fields.
+
+- `id` <[Number]> Identifier for this item in the manifest, used to lookup its concrete identifier for coding.
+- `name` <[String]> Name of the pick list choice.
+
+## Statistics
+Statistics are counters that Ringtail will automatically aggregate across cases and roll up into queryable summaries at the portal. These are great for tracking things like feature usage and billing metrics, such as "count of docs processed" or "characters translated".
```

Comment:
What about "number of characters translated"? (Or number of words; I think that's how human translators work.) Also, move the period to inside the quotation marks.